### PR TITLE
chore: usage of goerli removed

### PR DESCRIPTION
**Pull request by saledjenic**
Corresponding `status-go` PR:
- https://github.com/status-im/status-go/pull/5945

For the testing purpose this PR points to `status-go` commit `5a19cc2a6ebc23d5dd8fcac2530a2897b0443e42 ` that also includes changes done in:
- https://github.com/status-im/status-go/pull/5924
- https://github.com/status-im/status-go/pull/5957

### CODE REVIEW

**Review by virginiabalducci — APPROVED** [review]
Looks good! 
I checked the console log and the App's advanced settings and there is no sign of Goerli.

File: libs/Onboarding/src/Onboarding/Common/Constants.h
Comment (alexjba) [inline libs/Onboarding/src/Onboarding/Common/Constants.h:1]: @iurimatias @alaibe  Any idea if we have some plans with this c++ app? Maybe it's time to clean the repo

**Review by alexjba — APPROVED** [review]
LGTM!

File: libs/Onboarding/src/Onboarding/Common/Constants.h
Comment (caybro) [reply libs/Onboarding/src/Onboarding/Common/Constants.h:1]: Yeah, on my TODO :) Together with the sandbox app

**Review by caybro — COMMENTED** [review]
LGTM

We also have a `NetworksModel.qml` in storybook, could you pls remove the Goerli stuff from there too?

**Review by caybro — APPROVED** [review]
Thanks 

Now you got some legit failures in tst_SwapModal :grimacing: sorry

**Comment by virginiabalducci** [comment]
The approval was only made for the removal of goerli part, I still need to test the scope of https://github.com/status-im/status-go/pull/5924

**Review by micieslak — APPROVED** [review]
LGTM!

**Pull request merged**